src/control/settings/ButtonConfig.cpp

```diff
@@ -50,18 +50,18 @@ void ButtonConfig::applyConfigToToolbarTool(ToolHandler* toolHandler) {
     toolHandler->selectTool(this->action);
 
     if (this->action == TOOL_PEN || this->action == TOOL_HIGHLIGHTER || this->action == TOOL_ERASER) {
```

Comment:
@rolandlo : This part I only changed to make the flow similar to `ButtonConfig::applyNoChangeSettings`, which was the function that actually needed to change.
So in this function the changes should be logically equivalent to the version before.